ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -571,4 +533,10 @@ private class TableViewHeaderFooterTitleView: UITextView {
             // It should always return nil which indicates there is no current selection (https://developer.apple.com/documentation/uikit/uitextinput/1614541-selectedtextrange)
         }
     }
+
+    var linkColor: UIColor? {
```

Comment:
Maybe this could be a lazy non-optional var, and have a default value of `.brandForeground1` instead of needing to check for nil in `updateLinkTextColor()`, but I also don't love needing to have a default both here and in the header footer's token set.

Reply:
Could maybe also just rewrite the init, since it's only being used in this class, to just take a link color. We're not passing in a frame or text container, anyway, it's getting default values of `.zero` and `nil`.

Reply:
Oh, but then `titleView` would have to be lazy.

Reply:
The change was simple enough so I made it happen!

Reply:
Oh I only saw the first message, but either way it should work with theme changes too